core/storage/email/gae_models.py

```diff
@@ -275,6 +293,19 @@ def get_deletion_policy():
         """Sent email should be kept for audit purposes."""
         return base_models.DELETION_POLICY.KEEP
 
+    @classmethod
+    def has_reference_to_user_id(cls, user_id):
+        """Check whether BulkEmailModel exists for user.
+
+        Args:
+            user_id: str. The ID of the user whose data should be checked.
+
+        Returns:
+            bool. Whether any models refer to the given user ID.
+        """
+        # TODO(vojtechjelinek): Add check for recipient_ids too.
```

Comment:
Create an issue and use `TODO(#ISUUE_ID): description` pattern.

Reply:
I will remove the whole method for now.